mozilla-release/browser/base/content/aboutDialog.css

```diff
@@ -11,7 +11,8 @@
 }
 
 #rightBox {
-  background-image: url("chrome://branding/content/about-wordmark.svg");
+  /* CLIQZ. Don't use
+  background-image: url("chrome://branding/content/about-wordmark.svg");*/
```

Comment:
Please check ticket DB-1262, there are an image with it. It is out of our design for About dialog